zaza/openstack/charm_tests/test_utils.py

```diff
@@ -206,13 +206,18 @@ def restart_on_changed(self, config_file, default_config, alternate_config,
         logging.debug('Remote unit timestamp {}'.format(mtime))
 
         with self.config_change(default_config, alternate_config):
-            logging.debug(
-                'Waiting for updates to propagate to {}'.format(config_file))
-            model.block_until_oslo_config_entries_match(
-                self.application_name,
-                config_file,
-                alternate_entry,
-                model_name=self.model_name)
+            # If this is not an OSLO config file set default_config={}
+            if alternate_entry:
```

Comment:
Isn't this covered by the test by test_110_restart_on_config_change

Reply:
This change allows for  non-oslo config file